### ISSUE TITLE
leaflet-FreeDraw for angular 4

### ISSUE THREAD

**Issue by Zahma** [Wildhoney/Leaflet.FreeDraw#123]
Hello I'm using the following module in angular 4:

https://github.com/Wildhoney/Leaflet.FreeDraw

in the following way:

import { FreeDraw } from 'leaflet-freedraw';

The project is compiling, which means that there is no problem of typings.

However, I'm getting the following error in the console:

Uncaught ReferenceError: regeneratorRuntime is not defined

I tried another solution, which is to import the project directly as JS, and write the typings as follow:

import 'leaflet-freedraw/dist/leaflet-freedraw.web.js';
import { FreeDraw } from '../xxx/freeDraw-typings';
with freeDaw-typings containing the following:

export class FreeDraw {
    constructor(params?: any);
    on(params: any):any;
    mode(params: any):any;
    create(params: any):any;
}
but I get the following:

    ERROR in ./src/main/webapp/app/xxxxx/xxxx/xxx.component.ts
Module not found: Error: Can't resolve '../xxx/freeDraw-typings'

**Comment by Wildhoney** [comment]
For the `regeneratorRuntime` issue you will need to bundle [`https://babeljs.io/docs/en/babel-polyfill.html`](`babel-polyfill`) with your build 👍

**Comment by Zahma** [comment]
@Wildhoney  thank you for the answer, you mean that I should include babel-polyfill in my packages ?

**Comment by Wildhoney** [comment]
Yep, and then if you're using Webpack you can do something like:

```javascript
module.exports = {
  entry: ["babel-polyfill", "./app.js"]
};
```

**Comment by Zahma** [comment]
@Wildhoney  thank you for your help, in fact I did it as you said, the library is apparently well imported. Still, I get a weired error: 
`ERROR TypeError: leaflet_freedraw_1.FreeDraw is not a constructor
`
Knowing that Accessing FreeDraw throgh window['FreeDraw'] is displaying the object.

**Comment by glau-isidoro** [comment]
@Zahma I'm getting the same error.
